2-Regression/1-Tools/translations/README.az.md

```diff
@@ -0,0 +1,227 @@
+- **sklearn**. Bu isə [Scikit-learn](https://scikit-learn.org/stable/user_guide.html) kitabxanasıdır.
+
+Tapşırıqda sizə kömək olması üçün kitabxanalardan bəzilərini köçürün.
+
+1. Aşağıdakı kodu yazaraq köçürmələri edək:
+
+   ```python
+   import matplotlib.pyplot as plt
+   import numpy as np
+   from sklearn import datasets, linear_model, model_selection
+   ```
+
+   Yuxarıda siz `matplotlib`-i, `numpy`-ı və `sklearn`-dən `datasets`, `linear_model` və `model_selection`-ı köçürürsünüz. `model_selection` datanı öyrədilmə və test massivlərinə bölmək üçün istifadə olunur.
+
+### Şəkərli diabet data massivi
+
+The built-in [diabetes dataset](https://scikit-learn.org/stable/datasets/toy_dataset.html#diabetes-dataset) includes 442 samples of data around diabetes, with 10 feature variables, some of which include:
+
+Öncədən qurulmuş olan [şəkərli diabet data massivinə](https://scikit-learn.org/stable/datasets/toy_dataset.html#diabetes-dataset), 10 xüsusiyyət dəyişəni olmaqla, bu xəstəliklə bağlı 442 nümunə daxildir. Xüsusiyyət dəyişənlərinin bəziləri aşağıdakılardır:
+- age: illər ilə yaş
+- bmi: bədənin çəki indeksi
+- bp: ortalama qan təzyiqi
+- s1 tc: T-hüceyrələri(ağ qan hüceyrələrinin növü)
+
+✅ Bu data massivinə, şəkərli diabet haqqında araşdırma etmək üçün önəmli bir xüsusiyyət dəyişəni olaraq 'cins' anlayışı da daxildir. Əksər tibbi data massivlərinə bu tip ikili sinifləndirmə daxil edilir. Bir qədər bu formada kateqoriyalaşdırılmanın əhalinin bir qismini müalicələrdən necə kənarda tuta biləcəyi barəsində düşünün.
+
+İndi isə, X və y datalarını yükləyək.
+
+> 🎓 Yadda saxlayın ki, bu nəzarətli öyrənmə olduğuna görə 'y' adlı hədəfə ehtiyacımız var.
+
+Yeni kod xanasında, şəkərli diabet data massivini `load_diabetes()`-i çağıraraq yükləyin. Verilən `return_X_y=True`, `X`-in bir data matrisi, `y`-in isə reqressiya hədəfi olacağını göstərir.
+
+1. Data matrisinin formasını və ilk elementini göstərmək üçün bir neçə "print" komandası əlavə edin:
+
+    ```python
+    X, y = datasets.load_diabetes(return_X_y=True)
+    print(X.shape)
+    print(X[0])
+    ```
+
+    Cavab olaraq aldığınız şey, tuple adlanır. Etdiyiniz isə, tuple-ın ilk iki dəyərini sıra ilə `X` və `y`-ə mənimsətməkdir. Tuple haqqında ətraflı [buradan](https://wikipedia.org/wiki/Tuple) öyrənin.
+
+    Gördüyünüz kimi bu məlumat, 10 elementli massivi olan 442 bənddən ibarətdir:
+
+    ```text
+    (442, 10)
+    [ 0.03807591  0.05068012  0.06169621  0.02187235 -0.0442235  -0.03482076
+    -0.04340085 -0.00259226  0.01990842 -0.01764613]
+    ```
+
+    ✅ Data və reqressiya hədəfi arasındakı əlaqə barəsində biraz düşünün. Xətti reqressiya X xüsusiyyəti ilə y dəyişəni arasındakı əlaqəni təxmin edir. Dokumentasiyada şəkərli diabet data massivindəki [hədəfi](https://scikit-learn.org/stable/datasets/toy_dataset.html#diabetes-dataset) tapa bilərsiniz? Bu data massivi hədəf nəzərdə tutulduqda, nəyi təsvir edir?
```

Comment:
Dokumentasiyada - can be replaced with `Sənəddə`? Same applies for the previous usage

Reply:
I think that, we should use more common version of words with english. To be honest, none of using "sənəd" instead of "dokumentasiya". By time, our language will be made to adopt these kind of global terms.